jaeger-zipkin/build.gradle

```diff
@@ -1,12 +1,12 @@
 description = 'Integration library for Zipkin'
 
 dependencies {
+    compile project(':jaeger-client')
+    compile project(path: ':jaeger-thrift', configuration: "default")
```

Comment:
For some reason, we need to use the `default` configuration here to get access to the Thrift-generated classes (the ones under `gen-java`).